cmd/web/main.go

```diff
@@ -54,7 +54,7 @@ func openDB(dsn string) (*sql.DB, error) {
 func main() {
 	// configuration
 	addr := flag.String("addr", ":4000", "HTTP network address")
-	dsn := flag.String("dsn", "dev:Akinyemi1234@@/snippetbox?parseTime=true", "MySQL data source name")
+	dsn := flag.String("dsn", "root:Akinyemi1234@@/snippetbox?parseTime=true", "MySQL data source name")
```

Comment:
@ChrisRoland You changed the access level here from dev to root. This gives infinite access to malicious hacks on the DB. Fix this.

Reply:
Resolved.